index.js

```diff
@@ -137,6 +140,49 @@ function displayCategories() {
   });
 }
 
+function displayUsers() {
+  const users = forumData["users"];
+  const ids = users.map((user) => user.id);
+
+  let onlineUsersAvatars = `<span>Online (${ids.length}):</span>`;
+  ids.forEach((id) => {
+    onlineUsersAvatars += getUserAvatarComponent(id);
+  });
+
+  userListContainer.innerHTML = onlineUsersAvatars;
+}
+
+//handles the collapsing/expanding transitions
+function activeCollapsableUserList() {
```

Comment:
Hey,

great job again, I can see you have some experience writing code. 😸 

Couldn't we just replace this function with CSS property `transition-delay: 1s;` instead?

Either way, it's good to be merged. If you think the CSS property is better solution, you can change it. If not, could you provide an explanation why not?

Thank you

Reply:
You are right! I didnt think about that, i replaced the function, thank you for the feedback!